### ISSUE TITLE
Thank you!

### ISSUE THREAD

**Issue by Gitoffthelawn** [Infocatcher/Right_Links_WE#1]
Thank you for working on this! :)

I'm trying to understand the README.  Given the technology available in FF57 (now in Beta channel), is basic functionality possible?

**Comment by Infocatcher** [comment]
Basic functionality is already implemented, so it's possible to open regular links from content pages in new tabs. But currently hardcoded opening in sibling tab (no API to read user preferences and no easy way to re-implement browser.tabs.insertRelatedAfterCurrent = true behavior).
It's (currently?) impossible to handle non-content things (bookmarks, history items), also doesn't work well with void and dummy links with click handlers (e.g. `<a href="#" onclick="return doSomething();">click me</a>`). And doesn't work with `<canvas>` images (due to security restrictions).

**Comment by Gitoffthelawn** [comment]
Thanks so much for the explanation.  I'm not seeing the WE version on AMO.  Is it in the queue yet?

**Comment by vurtomatic** [comment]
Would love to have the basic functionality of right clicking on links on content pages, that's the most used imo. Currently using Tap To Tab but that's a double-click instead of one right click!

**Comment by MOFTAD** [comment]
@Gitoffthelawn Download the ZIP file, extract it, run the _make_ batch file, and install the resultant XPI file.

**Comment by vurtomatic** [comment]
@MOFTAD Is there a way to run the make batch file on a Mac? Or can someone compile this into an addon?

**Comment by Infocatcher** [comment]
May be <a href="https://github.com/Infocatcher/Right_Links_WE/blob/master/make.sh">make.sh</a> will work on Mac too: https://github.com/Infocatcher/Right_Links_WE/commit/20bd80c015dce9cccca10ce1421af446f6f49b1b

Anyway, this is just a dummy way to create ZIP archive with all source files (with some exceptions like not needed \*.md).

**Comment by Infocatcher** [comment]
https://github.com/Infocatcher/Right_Links_WE/releases/tag/0.1a1

**Comment by vurtomatic** [comment]
@Infocatcher thanks for the 0.1a1! But Firefox wouldn't let it install, says "this add-on could not be installed because it has not been verified".

**Comment by Infocatcher** [comment]
To try: about:debugging -> Load Temporary Add-on
Or in Nightly still available xpinstall.signatures.required = false (about:config)

**Comment by vurtomatic** [comment]
> To try: about:debugging -> Load Temporary Add-on

That worked, thank you!

**Comment by Infocatcher** [comment]
Signed beta version:
https://addons.mozilla.org/addon/right-links/versions/0.5b1
Note: changed ID of extension (now the same as old Right Links), so previous WE version should be uninstalled.

**Comment by Marcosdw** [comment]
Thank you upgraded this extension for Webextension ;)

**Comment by Gitoffthelawn** [comment]
@Infocatcher WE version no longer available on AMO.  Do you know what happened?  Can you get it back on AMO?

**Comment by Infocatcher** [comment]
> The excellent WE version is no longer available on AMO. Do you know what happened? Can you get it back on AMO?

All beta versions were removed from AMO side (were hidden and only author may see them)...
https://blog.mozilla.org/addons/2018/02/28/discontinuing-support-for-beta-versions/
Signed versions available here: https://github.com/Infocatcher/Right_Links_WE/releases
So, I should re-upload some new version to AMO (not user and author-friendly).

**Comment by Gitoffthelawn** [comment]
I agree, that although it's not ideal in many ways, a new version on AMO is the best for the future of your fantastic extension.

**Comment by Yaron10** [comment]
Thank you for making Right Links compatible with FF Quantum.
Great work. :+1:
